backend/data_preparation/crawler/twitter_crawler.py

```diff
@@ -81,6 +61,16 @@ def crawl(self, keywords: List, batch_number: int, fetch_from_db: bool) -> Union
 
             # gets status with the list that has batch number (can be a bit more than the batch#) amount of tweets
             ids = list(self.crawled_id_set)
+        elif using_sample_api:
+            ids = []
```

Comment:
maybe better to use a set to avoid duplicate id that matches for multiple keywords.